app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -130,7 +134,18 @@ export class FiltersControl extends Component {
 
   componentDidMount() {
     const { page } = this.state;
+
     this.fetchFilter({ page });
+    this.props.activeLaunchFilter && this.applyDefaultFilter();
+  }
+
+  componentDidUpdate(prevProps) {
```

Comment:
I think it's unnecessary to apply default filter again when filters will be fetched. We can do it once in `componentDidMount`.
So please remove this method